Integrations/CounterCraft/CounterCraft.yml

```diff
@@ -0,0 +1,419 @@
+commonfields:
+  id: CounterCraft Deception Director
+  version: -1
+name: CounterCraft Deception Director
+display: CounterCraft Deception Director
+category: Deception
+description: CounterCraft Deception Solution detects advanced adversaries. Automate
+  counterintelligence campaigns to discover targeted attacks with real-time active
+  response.
+configuration:
+- display: Deception Director Domain or IP Address
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Fetch incidents
```

Comment:
please place the fetch params below the others

Reply:
I don't know what you mean. Can you explain further?